docs/reference/cat/nodeattrs.asciidoc

```diff
@@ -1,8 +1,68 @@
 [[cat-nodeattrs]]
 === cat nodeattrs
 
-The `nodeattrs` command shows custom node attributes.
-For example:
+Returns information about custom node attributes.
+
+[[cat-nodeattrs-api-request]]
+==== {api-request-title}
+
+`GET /_cat/nodeattrs`
+
+[[cat-nodeattrs-api-query-params]]
+==== {api-query-parms-title}
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=http-format]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=cat-h]
++
+--
+If no column values are provided, the default columns are returned in the order
+below. If any column value is provided, no default columns are returned and
+must be explicitly requested instead.
+
```

Comment:
I'm tripping over "column values" a little--it feels like it should either just be columns or column headers. (Values feels like the attribute values, rather than heading identifiers.) I'd probably go with something like:

> If you do not specify which columns to include, `_cat/nodeattrs` returns the default columns in the order listed below. If you explicitly specify one or more columns, it returns only the specified columns.

(I haven't looked, but this sounds like standard boiler-plate that we might want to tweak.)

Reply:
Thanks for the suggestion @debadair. I think I was overthinking this one and the wording got awkward.

I implemented your changes with bdc833e.